### DISCUSSION TITLE
Cannot import name 'RotatE' from 'torch_geometric.nn'

### DISCUSSION THREAD

**Discussion by songsong0425** [pyg-team/pytorch_geometric#7329] [Q&A] [answered]
Dear PyG community,
Hi, I have a question about running the knowledge graph embedding models using PyG.
(Reference: https://github.com/pyg-team/pytorch_geometric/blob/master/examples/kge_fb15k_237.py)
When I tried to run the Python file, there was ImportError due to the absence of RotatE in torch_geometric.nn.
Just before, I faced a similar error with the fb15k_237 dataset from dataset /__init__.py file and solved it by upgrading the Python package.
So I checked the original GitHub page(https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/__init__.py), but I couldn't find the RotatE anywhere.
How can I deal with this problem?

**Comment by wsad1** [comment, marked as answer]
`RotatE` is present in `https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/kge/rotate.py`. Which version of PyG are you using. Updating to the latest PyG version should resolve this: https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html.

**Reply by songsong0425** [reply]
Thank you for your reply! I'm using torch_geometric 2.3.1. Isn't it the latest version?

**Reply by songsong0425** [reply]
I solved the problem by manually copy and paste the rotate.py and __init__.py in the corresponding directory.
